runtime/src/main/java/org/corfudb/runtime/collections/CorfuCounter.java

```diff
@@ -0,0 +1,26 @@
+package org.corfudb.runtime.collections;
```

Comment:
I see. This class seems to be obsolete. Please remove it.

Reply:
Sorry looks like this class got re-added when I did a rebase... It is not used anywhere and will be removed in https://github.com/CorfuDB/CorfuDB/pull/1882 which depends on this PR